.gitignore

```diff
@@ -0,0 +1,199 @@
+# IntelliJ project files
+.idea
+*.iml
+out
+gen
```

Comment:
wat, this file seems a bit insanely huge. do we actually need to ignore any of this stuff?

Reply:
oooh. I'm using idea's .gitignore template